dora/src/driver/start.rs

```diff
@@ -103,31 +103,36 @@ pub fn start(content: Option<&str>) -> i32 {
     code
 }
 
+const STDLIB: &[(&str, &str)] = &include!(concat!(env!("OUT_DIR"), "/stdlib.rs"));
```

Comment:
Could you please rename this file to something else, e.g `dora_stdlib_bundle.rs`? We already have a file `stdlib.rs`, this might otherwise be confusing...